.changeset/fifty-laws-sparkle.md

```diff
@@ -0,0 +1,8 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/core-typings": patch
+"@rocket.chat/model-typings": patch
+"@rocket.chat/models": patch
+---
+
+Now all the cloud credentials are stored in its own collection, preventing stale tokens from being used
```

Comment:
I think we could explain a bit better

The fact is not that a new collection prevents this, but not using the cache, which can have a minimal delay, is what fixes the problem